lib/src/models/users.dart

```diff
@@ -0,0 +1,21 @@
+class Users {
```

Comment:
it's best to have models in singular - i.e. `User`

Reply:
ok. i will change

Reply:
done.